server/graphql/v2/interface/Account.js

```diff
@@ -143,6 +143,10 @@ const accountFieldsDefinition = () => ({
         type: new GraphQLList(AccountType),
         description: 'Type of accounts (BOT/COLLECTIVE/EVENT/ORGANIZATION/INDIVIDUAL)',
       },
+      orderBy: {
+        type: new GraphQLNonNull(ChronologicalOrderInput),
```

Comment:
It should be nullable.

Reply:
I'm personally not clear on the behavior between `GraphQLNonNull` and `defaultValue`, so I'm gonna trust you.

Reply:
When I made it non-nullable it threw more errors.
<img width="1778" alt="Screenshot 2020-08-24 at 18 08 09" src="https://user-images.githubusercontent.com/37520401/91074709-d84f8780-e634-11ea-9620-91ccaae95645.png">

Reply:
> When I made it non-nullable it threw more errors.

@sbinlondon You mean, when you made it nullable?